cluster/ClusterClient.java

```diff
@@ -42,9 +42,10 @@
 import static grakn.common.collection.Collections.pair;
 
 public class ClusterClient implements GraknClient.Cluster {
+
     private static final Logger LOG = LoggerFactory.getLogger(ClusterClient.class);
     private final Map<String, CoreClient> coreClients;
-    private final Map<String, GraknClusterGrpc.GraknClusterBlockingStub> graknClusterRPCs;
+    private final Map<String, ResilientStub.Cluster> blockingStubs;
```

Comment:
I like the fact that `ResilientStub` has been introduced. Yet I think we can actually do better than the name `ResilientStub`.

What actually is the purpose of `ResilientStub`? A stub that is resilient, or an API for accessing Grakn Protocol service methods?

I'd say the implementation is a resilient stub, the purpose is a Grakn service.

So how about we rename it to `GraknService`, or `GraknProtoService`, or `ProtoService`, or ...

Reply:
👍 we've renamed it to GraknStub now